Reference/Config/tinyMceConfig/index.md

```diff
@@ -1,82 +1,118 @@
 ---
-versionFrom: 7.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
-# tinyMceConfig
+# tinyMceConfig - Rich Text Editor Configuration
 
-Here you will find documentation relating to the options you can add/modify in the  tinyMceConfig.config file. The configuration you set here will be used by each TinyMce instance as it is initialized in Umbraco.
+The Rich Text Editor in the Umbraco V8 Backoffice utilises the rich text editor plugin: [TinyMce V4.x](https://www.tiny.cloud/docs-4x/). This configuration file **'/config/tinyMce.config'** is used to set globally every TinyMce instance when it is intialized by Umbraco. Not all TinyMce features are enabled by default in Umbraco, you can use this configuration file to enable other TinyMce plugins or ammend the core behaviour of the Rich Text Editor in Umbraco.
 
-## commands
+## Commands / Toolbar
 
-Inside of the `<commands>` node you will find multiple `<command>` nodes. Each one of these nodes defines a icon/button that can appear on the formatting bar inside of Umbraco and the command that is triggered when it is clicked.
+Inside the `<commands>` section of the configuration file you will find multiple `<command>` nodes. Each one of these configuration elements define an 'icon/button' option that can appear on the formatting toolbar of the Rich Text Editor instance. When configuring a rich text editor Data Type instance you can pick and choose the 'commands/toolbar icons' that should be included.
 
 ```xml
-<command>
-  <umbracoAlias>code</umbracoAlias>
-  <icon>images/editor/code.gif</icon>
-  <tinyMceCommand value="" userInterface="false" frontendCommand="code">code</tinyMceCommand>
-  <priority>1</priority>
-</command>
+        <command alias="cut" name="Cut" mode="Selection"/>
+        <command alias="copy" name="Copy" mode="Selection"/>
+        <command alias="paste" name="Paste" mode="All" />
 ```
 
-`umbracoAlias` defines a unique alias within Umbraco for the command. This alias should not contain any spaces.
+`alias` defines a unique alias within Umbraco for the command. This alias should not contain any spaces.
 
-`icon` defines the path to an image file to be used on the formatting toolbar. This image should be 16px x 16px in size.
+`name` defines the name that will be seen in the list of Toolbar configuration options in the backoffice for the rich text editor 
 
-`tinyMceCommand` defines the tinyMceCommand properties.
+`mode` this can be set to be either 'Insert', 'Selection' or 'All'. In 'distraction free' mode, there are two 'quick menus' that appear - one when you are about to 'insert' text, and one when you 'select' existing text. This setting determines which 'quick menu' the icon should appear on or whether it should appear on both!
```

Comment:
```suggestion
`mode` - This can be set to be either 'Insert', 'Selection', or 'All'. In 'distraction free' mode, there are two 'quick menus' that appear - one when you are about to 'insert' text, and one when you 'select' existing text. This setting determines which 'quick menu' the icon should appear on or whether it should appear on both.
```